analysis_options.yaml

```diff
@@ -0,0 +1,356 @@
+analyzer:
+  exclude:
+    - "**/*.g.dart"
+    - "**/*.gform.dart"
```

Comment:
instead of copying analyzer file, can you just depend on `lint_quido`, it will be easier to maintain sync across packages? WDYT?

Reply:
Replaced to use `lint_quido`